net/safe-search-bing/Makefile

```diff
@@ -0,0 +1,60 @@
+#
+# Copyright (c) 2018 Gregory L. Dietsche (Gregory.Dietsche@cuw.edu)
+# This is free software, licensed under the MIT License
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=safe-search-bing
+PKG_VERSION:=1.0.0
+PKG_RELEASE:=4
+PKG_LICENSE:=MIT
+PKG_MAINTAINER:=Gregory L. Dietsche <Gregory.Dietsche@cuw.edu>
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/safe-search-bing
+	SECTION:=net
+	CATEGORY:=Network
+	TITLE:=Block explicit images, videos, and websites from Bing Search results.
+	PKGARCH:=all
```

Comment:
do we need this?

Reply:
I'm not sure what you mean...

Reply:
If you meant the TITLE, I've updated it for both packages to be short and to the point.

Reply:
I meant the PKGARCH, as best I can tell, you don't need it.

Reply:
I just tried doing a build without PKGARCH set. It does work, but seems to create architecture specific packages even though that's not needed. So it seems to me that we should leave PKGARCH:=all in for the most flexibility, but I'll defer to the experts if I'm wrong.

Here's what I'm seeing...

Without PKGARCH:

    ~# opkg info safe-search-google
    Package: safe-search-google
    Version: 1.0.0-1
    Depends: libc
    Status: unknown ok not-installed
    Section: net
    Architecture: all
    Size: 4288
    Filename: safe-search-google_1.0.0-1_all.ipk

With PKGARCH:=all

    ~# opkg info safe-search-google
    Package: safe-search-google
    Version: 1.0.0-1
    Depends: libc
    Status: unknown ok not-installed
    Section: net
    Architecture: mips_24kcSize: 4295
    Filename: safe-search-google_1.0.0-1_mips_24kc.ipk

Thanks!